metadata-db-app/src/cmr/metadata_db/data/memory_db.clj

```diff
@@ -202,15 +202,16 @@
   (get-granule-concept-ids
    [this provider native-id]
    (let [provider-id (:provider-id provider)
-         concept-ids-fn (fn [gran]
-                          [(:concept-id gran) (get-in gran [:extra-fields :parent-collection-id])])]
-     (->> @concepts-atom
-          (filter (fn [c]
-                    (and (= :granule (:concept-type c))
-                         (= provider-id (:provider-id c))
-                         (= native-id (:native-id c)))))
-          first
-          concept-ids-fn)))
+         matched-gran (->> @concepts-atom
```

Comment:
Nitpick: I think indentation is a little off here

Reply:
It is indented correctly based on my ATOM editor.